CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+You want to help? You rock! Now, take a moment to be sure your contributions make sense to everyone else.
```

Comment:
Any project with an issue tracker deserves it, especially one that has had activity. It shows up automatically in the issue tracker and many developers actually find it very helpful.